docs/modules/ROOT/pages/reference/index.adoc

```diff
@@ -56,9 +56,85 @@
 |Stable
 |1.5.11
 |
+
 // standards: END
 |===
 
+[[unsupported]]
+== Unsupported Modules
+
+|===
+| Quarkus CXF extension | Alternative
+
+| JAX-RS +
+`cxf-rt-frontend-jaxrs` `cxf-rt-rs-client`
+| https://quarkus.io/guides/rest-json[Use quarkus core rest-json support] 
+
+| https://cxf.apache.org/fediz.html[Fediz]
+| legacy SAML protocol is not supported but openId connect is supported on Quarkus Core
+
+| https://cxf.apache.org/docs/aegis-21.html[Aegis] 
+| use JAXB or JAXWS
+
+| https://cxf.apache.org/dosgi-architecture.html[DOSGI] +
+Karaf
+|none
```

Comment:
```suggestion
Karaf
| `quarkus-cxf`
```

Reply:
for this one I do not understand the change.

Reply:
I thought Karaf was an OSGi based container for web applications. So Quarkus and specifically Quarkus CXF could be seen as an alternative, no?